please delete this PR

azgaresncf opened the pull request:
please delete this PR

Commit:
adding cached consentframework and cookiebot domains

(fixing too much websites)

Commit:
adding expert.de and missing cached consentframework

Commit:
adding guide-outillage css rule

Commit:
adding gastronom.ru - fixes #8118

Commit:
adding forsal.pl - fixes #8114

Commit:
adding greekcity.fr - fixes #8116

Commit:
adding arduino.cc - fixes too much issues

fixes #7937, fixes #7783, fixes #7597, fixes #7589, fixes #7479, fixes #7361, fixes #7218, fixes #7038, fixes #6959, fixes #6858, fixes #6844, fixes #2803

Commit:
editing aliexpress rules - too many issues

- fixes #8112
- fixes #8073
- fixes #8052
- fixes #8027
- fixes #7963
- fixes #7929
- fixes #7912
- fixes #7873
- fixes #7840
- fixes #7837
- fixes #7547
- fixes #4862
- fixes #3800

Commit:
adding data-media and v2 - fixes #8111

Commit:
adding censor.net and setupcmp

Commit:
completely block usercentrics

Commit:
changing cookiebot

GrayedFox reviewed (approved):
Looks good to me 👍🏽

Commit:
adding various sites rules

komoot.de: closes #6337, closes #6386, closes #6531, closes #6561, closes #6624, closes #6792, closes #7125, closes #7424, closes #7642, closes #7869, closes #8535, closes #8552, closes #9100, closes #9805

bauer-baumschulen.ch: closes #9802

allyouplay.com: closes #6418, closes #9801

woojer.com: closes #9799

sede.coruna.gal: closes #9800

lasexta.com (sibbo CMP): closes #3368, closes #6416, closes #6446, closes #6583, closes #6893, closes #8219, closes #9798

holzkern.com: closes #3376, closes #6344, closes #8276, closes #9241, closes #9294, closes #9795

Commit:
fixed a mistake
